tests/scripts/src/test/bash/bk_test_bin_common.sh

```diff
@@ -32,7 +32,8 @@ testDefaultVariables() {
   assertEquals "BINDIR is not set correctly" "${BK_BINDIR}" "${BINDIR}"
   assertEquals "BK_HOME is not set correctly" "${BK_HOMEDIR}" "${BK_HOME}"
   assertEquals "DEFAULT_LOG_CONF is not set correctly" "${BK_CONFDIR}/log4j.properties" "${DEFAULT_LOG_CONF}"
-  assertEquals "NETTY_LEAK_DETECTION_LEVEL is not set correctly" "disabled" "${NETTY_LEAK_DETECTION_LEVEL}"
+  assertEquals "NETTY_LEAK_DETECTION_LEVEL is not set correctly" "disabled" "${NETTY_LEAK
```

Comment:
I am not sure why the changes in this file are needed. `4.8.0` here is just version number used in the testing code. the testing code isn't coupled with the build.

Reply:
I've no idea why this file is needed, I just changed every place I saw 4.8.0

Reply:
do you mind reverting this change back? because the change will become misleading. in future if similar problem happens, people might check the gitlog and follow what the change was doing here.

Reply:
sure. what is this file for?

Reply:
these are the tests for testing our bash scripts